theme.json

```diff
@@ -0,0 +1,413 @@
+          "line-height": "clamp(1.1em, calc(1.25em + -0.4vw), 1.2em)",
+          "font-size": "clamp(1.38rem, calc(1.08rem + 1.52vw), 2.64rem)",
+          "letter-spacing": "-0.01em"
+        },
+        "5": {
+          "font-family": "Aktiv Grotesk VF",
+          "font-weight": 500,
+          "line-height": "clamp(1.1em, calc(1.25em + -0.4vw), 1.2em)",
+          "font-size": "clamp(1.57rem, calc(1.04rem + 2.69vw), 3.81rem)",
+          "letter-spacing": "-0.01em"
+        }
+      },
+      "space": {
+        "padding": {
+          "global": "clamp(1.00rem, calc(0.40rem + 3.01vw), 3.50rem)"
+        },
+        "block": {
+          "gap": "clamp(1.50rem, calc(1.38rem + 0.60vw), 2.00rem)"
+        }
+      },
+      "alignment": {
+        "center": "744px",
+        "wide": "1008px"
+      },
+      "background": {
+        "primary": "#f5f2f1"
+      },
+      "text": {
+        "primary": "#13191e",
+        "secondary": "#444140",
+        "helper": "#6a6766",
+        "interactive": "#9a6014"
+      },
+      "link": {
+        "default": "#9a6014",
+        "hover": "#7a4909"
+      },
+      "button": {
+        "primary": {
+          "default": "#13191e",
+          "hover": "#d8a45f",
+          "active": "#ba7920",
+          "label": "#fcfafa"
+        }
+      }
+    },
+    "layout": {
+      "contentSize": "var(--wp--custom--alignment--center)",
+      "wideSize": "var(--wp--custom--alignment--wide)"
+    },
+    "spacing": {
+      "margin": false,
+      "padding": false,
+      "units": ["rem", "%"]
+    },
+    "typography": {
+      "customFontSize": false,
+      "fontStyle": false,
+      "fontWeight": false,
+      "letterSpacing": false,
+      "lineHeight": false,
+      "textDecoration": false,
+      "textTransform": false,
+      "dropCap": false,
+      "fontFamilies": [],
+      "fontSizes": []
+    },
+    "blocks": {
+      "core/heading": {
+        "typography": {
+          "fontSizes": [
+            {
+              "slug": "heading-1",
+              "size": "var(--wp--custom--heading--1--font-size)",
+              "name": "Heading 1"
+            },
+            {
+              "slug": "heading-2",
+              "size": "var(--wp--custom--heading--2--font-size)",
+              "name": "Heading 2"
+            },
+            {
+              "slug": "heading-3",
+              "size": "var(--wp--custom--heading--3--font-size)",
+              "name": "Heading 3"
+            },
+            {
+              "slug": "heading-4",
+              "size": "var(--wp--custom--heading--4--font-size)",
+              "name": "Heading 4"
+            },
+            {
+              "slug": "heading-5",
+              "size": "var(--wp--custom--heading--5--font-size)",
+              "name": "Heading 5"
+            }
+          ]
+        },
+        "color": {
+          "gradients": [],
+          "text": true,
+          "palette": [
+            {
+              "slug": "black",
```

Comment:
I go back and forth on whether it's better to obscure the token name from the user. So, calling it `black` instead of `primary`. Generally, I like keeping all involved parties on the same page, so when the user requests the developer to add a new type color they're calling it by the same name.

Reply:
That's a great point. I'll switch this to be in line with what the token is called rather than the underlying value of the token.